test/DynamoCoreWpfTests/WorkspaceOpeningTests.cs

```diff
@@ -96,6 +96,20 @@ public void OpeningWorkspaceDoNotStartWithUnSavedChanges()
         }
 
         [Test]
+        public void OpeningWorkspaceWithManualRunState()
+        {
+            var ws = (HomeWorkspaceModel)OpenWorkspaceInManualModeFromSampleFile(true);
+            Assert.IsFalse(ws.HasRunWithoutCrash);
+        }
+
+        [Test]
+        public void OpeningWorkspaceWithAutoRunState()
+        {
+            var ws = (HomeWorkspaceModel)OpenWorkspaceInManualModeFromSampleFile(false);
+            Assert.IsTrue(ws.HasRunWithoutCrash);
```

Comment:
shouldn't both these tests actually check the run type?

Reply:
I can definately add that check

Reply:
You could just add the check in the OpenWorkspaceInManualModeFromSampleFile right near the comment. It just makes sure that the assumption in the comment is true.